cmd/corectl/main.go

```diff
@@ -8,24 +8,30 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
 
 	"chain/core/accesstoken"
 	"chain/core/config"
+	"chain/core/fileutil"
 	"chain/core/migrate"
 	"chain/crypto/ed25519"
+	"chain/database/raft"
 	"chain/database/sql"
-	chainjson "chain/encoding/json"
 	"chain/env"
 	"chain/generated/rev"
 	"chain/log"
+	"chain/protocol/bc"
 )
 
 // config vars
 var (
-	dbURL = env.String("DATABASE_URL", "postgres:///core?sslmode=disable")
+	dbURL      = env.String("DATABASE_URL", "postgres:///core?sslmode=disable")
+	listenAddr = env.String("LISTEN", ":1999")
```

Comment:
My original plan had been to turn `corectl` into a client as a polish step, since I'm reluctant to pile more into this PR, and running this seems to work okay. BUT given that I was also thinking that the ability to remove a node from a cluster is also a polish step, I guess I should fix this (that is, turn corectl into a client) in this PR.